projects/storefrontlib/src/lib/user/checkout-login/checkout-login.component.html

```diff
@@ -0,0 +1,47 @@
+<form>
+  <div class="form-group">
+    <label>
+      <span class="label-content">Email address</span>
+      <input
+        type="email"
+        class="form-control"
+        formControlName="userId"
+        placeholder="Enter email"
+      />
+    </label>
+  </div>
+
+  <div class="form-group">
+    <label>
+      <span class="label-content">Confirm Email address</span>
+      <input
+        type="email"
+        class="form-control"
+        formControlName="userId"
```

Comment:
Could you change this formControlName so it's different to the first one please. So something like `userIdConf`

Reply:
done